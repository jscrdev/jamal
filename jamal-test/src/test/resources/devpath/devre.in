IF YOU SEE THIS  IN THE FILE devre.out THAT MEANS ERROR