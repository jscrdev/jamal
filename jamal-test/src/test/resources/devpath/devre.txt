
# this is a comment line       
/Users/verhasp/github/jamal/jamal-test/src/test/resources/devpath/devre.in=/Users/verhasp/github/jamal/jamal-test/src/test/resources/devpath/devre.re

