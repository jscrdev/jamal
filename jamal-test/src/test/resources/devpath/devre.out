pref

NOT THIS ONE

postf